carina-utils/src/main/java/com/qaprosoft/carina/core/foundation/utils/ZipManager.java

```diff
@@ -74,9 +70,7 @@ public static void unzip(String zip, String extractTo) {
             LOGGER.error(e);
         } finally {
             try {
-                if (zipFile != null) {
-                    zipFile.close();
-                }
+                zipFile.close();
```

Comment:
![BLOCKER](https://sonarsource.github.io/sonar-github/severity-blocker.png 'Severity: BLOCKER') NullPointerException might be thrown as 'zipFile' is nullable here [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://demo.qaprosoft.com/sonarqube/coding_rules#rule_key=squid%3AS2259)